### PR TITLE
Zendesk: Add requirements to ticket_forms, users, and organizations

### PR DESCRIPTION
**Pull request by dylan-stitch**
This PR:

- Adds info about the requirements for replicating `ticket_forms`
- Adds info about the requirements for replicating user and organization custom fields

### CODE REVIEW

File: _integration-schemas/zendesk/users.md [hunk 1]
```diff
@@ -288,4 +288,4 @@ attributes:
   - name: "verified"
     type: "boolean"
     description: "If `true`, the user's primary identity has been verified."
----
\ No newline at end of file
+---
```
Comment (dylan-stitch) [inline _integration-schemas/zendesk/users.md]: I'm not sure what's up with line 291. Is that expected?

File: _integration-schemas/zendesk/ticket_forms.md [hunk 2]
```diff
@@ -88,4 +88,4 @@ attributes:
       - name: "value"
         type: "integer"
         description: "The ID of the brand."
----
\ No newline at end of file
+---
```
Comment (dylan-stitch) [inline _integration-schemas/zendesk/ticket_forms.md]: I'm not sure what's up with line 91. Is that expected?

File: _integration-schemas/zendesk/organizations.md [hunk 3]
```diff
@@ -103,4 +103,4 @@ attributes:
   - name: "url"
     type: "string"
     description: "The API URL of the organization."
----
\ No newline at end of file
+---
```
Comment (dylan-stitch) [inline _integration-schemas/zendesk/organizations.md]: I'm not sure what's up with line 106. Is that expected?

Comment (erinkcochran87) [reply _integration-schemas/zendesk/organizations.md]: @dylan-stitch You mean just the three dashes? Yes. It's in the Metadocs as well, but [this is the official Jekyll documentation explaining it](https://jekyllrb.com/docs/front-matter/).

Basically, these dashes need to be present for Jekyll to detect these files. Without them, Jekyll won't throw an error - they'll just be ignored and therefore not rendered.

**Pull request merged**